src/modules/init/git.ts

```diff
@@ -2,16 +2,32 @@ import { copy, emptyDir, ensureDir, remove } from 'fs-extra'
 import pipeStreams from 'pipe-streams-to-promise'
 import request from 'request'
 import unzip from 'unzip-stream'
+import axios from 'axios'
+import log from '../../api/logger'
 
-const urlForRepo = (repo: string, org: string) => `https://github.com/${org}/${repo}/archive/master.zip`
+const urlForRepo = async (repo: string, org: string) => {
+  /**
+   * If we have internal templates in the future (in a private org repo) we have to figure out
+   * how to make this request authenticated. But, since toolbelt is open-source and so are the
+   * apps templates, we can afford not to take care of this right now.
+   */
+  try {
+    const { default_branch: branchName } = (await axios.get(`https://api.github.com/repos/${org}/${repo}`)).data
+    return { url: `https://github.com/${org}/${repo}/archive/${branchName}.zip`, branchName }
+  } catch (err) {
+    log.debug(err)
+    throw new Error(`We could not determine the default branch for repo ${org}/${repo}. Please try again.`)
```

Comment:
How about moving this string message to `Messages.ts` files ?  

Example: [here](https://github.com/vtex/toolbelt/blob/b4eba5ed9edbc6d67dcb3c55967905367e899787/src/api/modules/workspace/use.ts#L40)